doc/articles/toc.yml

```diff
@@ -14,6 +14,8 @@
       href: get-started-rider.md
     - name: dotnet new templates
       href: get-started-dotnet-new.md
+    - name: Get Started - Uno Figma
```

Comment:
Doesn't really fit here (despite having 'Get Started' in the title ;) ). Probably belongs under Development > Uno Features, alongside Uno Material/Lottie/etc.

Reply:
Agreed - it can be removed. We are creating Uno Platform for Figma specific documentation section. @duke7553 , could you please remove it?

Reply:
@sasakrsmanovic I've committed this one. Going to go ahead and suggest the rest of the changes.